Codeinwp/neve
# Hybrid FSE compatibility

### Summary
Adds compatibility for FSE in a hybrid mode. The PR should work with the child theme here: https://github.com/Codeinwp/neve-fse

After activating the child theme, a section should appear in the customizer, marked as "Beta". 

It adds a new Section in the customizer with toggles for the standard templates:
- Index;
- Front Page;
- Archive;
- 404;
- Search;
- Page;
- Single Post;

We provide some basic templates for all these things mentioned above inside the child theme mentioned above.



### Will affect visual aspect of the product

NO

### Screenshots 

<details><summary>Customizer Panel GIF</summary>

_Note: Some of the strings in the GIF below have been removed/changed to keep the translation impact at a minimum._

![fse-customize](https://user-images.githubusercontent.com/15010186/210397618-1be4f29f-be80-414a-8e48-afc17b892a44.gif)

</details>


### Test instructions


- Enable the child theme here: https://github.com/Codeinwp/neve-fse;
- Enable the FSE integration inside the customizer panel;
- Enable the different templates and check that they are loaded correctly;
- Check that header/footer toggles work fine;
- Check that the site editor shows the templates correctly and they are editable and render properly on the frontend;

### Known issues:
- Sometimes, depending on what templates are enabled and if the site uses a static front page or not, clicking the Site menu item will break the current screen
![image](https://user-images.githubusercontent.com/15010186/210398277-75f4687d-fc8b-49a5-ae31-6c4b0a708f6b.png)



Closes Codeinwp/neve-pro-addon#1583.
